### DISCUSSION TITLE
Cannot reverify academic affiliation

### DISCUSSION THREAD

**Discussion by pufeili** [community/community#56051] [GitHub Education]
### Hi everyone,

**It's strange that my student certification hasn't expired yet, but I can't use the student discount anymore. When I re authenticate, the following information is displayed:**
 > You were last verified as a student on . It is not necessary for you to re verify at this time.

May I ask if anyone can help me, plz?

**Comment by mickeradman** [comment]
I have a question on the same topic (almost).

I can't reverify because I use the same document as I did last time. But that one says I'm studying until 2023-06-09, and that fact hasn't changed.

Why do I need a NEW document?
